_includes/sqlstyle.guide.md

```diff
@@ -170,22 +151,6 @@ spacing is used. Do not crowd code or remove natural language spaces.
 
 #### Spaces
 
-Spaces should be used to line up the code so that the root keywords all end on
-the same character boundary. This forms a river down the middle making it easy for
-the readers eye to scan over the code and separate the keywords from the
-implementation detail. Rivers are [bad in typography][rivers], but helpful here.
```

Comment:
i like _looking_ at this, but I'm never going to write this. probably ever.

can you write up the alternative, instead of just deleting and leaving it ambiguous?